src/plugins/intel_cpu/src/nodes/conv.cpp

```diff
@@ -1119,6 +1058,14 @@ bool Convolution::isPossibleToSkipInitConfig(DnnlDesriptor &desc) const {
 
 std::shared_ptr<MemoryDesc> Convolution::getSrcMemDesc(dnnl::primitive_desc_iterator &primitive_desc_it, size_t idx) {
     auto desc = idx > 0 ? primitive_desc_it.weights_desc(idx - 1) : primitive_desc_it.src_desc(idx);
+
+    if (idx == 1 && constWeightMemPtr) {
+        // original weight memory is const node, we report same descriptor so no extra reorder
+        // will be inserted by graph, so later prepareParam() can prepare Weight from original
+        // const weight node
+        return constWeightMemPtr->getDescPtr();
+    }
```

Comment:
`constWeightMemPtr` was retrieved from the constant node rather than edge, so it's not `nullptr`, but you are right, we should return `plain` descriptor in all cases which is more consistent with deconv/fullyconnect's design